pkg/env/prod.go

```diff
@@ -117,6 +122,8 @@ func newProd(ctx context.Context, log *logrus.Entry, instancemetadata instanceme
 	p.clustersGenevaLoggingPrivateKey = clustersGenevaLoggingPrivateKey
 	p.clustersGenevaLoggingCertificate = clustersGenevaLoggingCertificates[0]
 
+	p.e2estorage = storage.NewAccountsClient("33cfc4e8-e3ac-4b48-8352-914daac4f924", rpAuthorizer)
```

Comment:
if you use rpAuthorizer, you'll need to grant rights on the storage account for every new region that's deployed.  Can you use p.FPAuthorizer() instead?